src/PrMerger.js

```diff
@@ -6,6 +6,18 @@ const GH = require('./GitHubUtil.js');
 const Util = require('./Util.js');
 const MergeContext = require('./MergeContext.js');
 
+class PrScanResult {
+    constructor() {
+        this.scanDate = new Date(); // the scan starting time
+        this.delay = null; // the next scan should start in this number of milliseconds
+        this.unchangedPrs = []; // PRs that may be ignored in the next scan
```

Comment:
To simplify and clarify (user) logic, please store all non-delayed PRs in this array:

```suggestion
        this.awakePrs = []; // PRs that were not delayed during the scan
```